dojo/db_migrations/0047_jira_minimum_severity_default.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.14 on 2020-07-10 08:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    def set_default_to_low(apps, schema_editor):
+        system_settings = apps.get_model('dojo', 'system_settings')
+        try:
+            ss = system_settings.objects.get(id=1)
+            jira_sev_value = ss.jira_minimum_severity
+            if jira_sev_value is None:
+                ss.jira_minimum_severity = 'Low'
+                ss.save()
+        except Exception as e:
+            # probably a test run such as running unittest, no values in table
+            print("No data migration because of exception caught: {}".format(e))
```

Comment:
I think changing existing installs is a best effort anyway so suggest to not log anything in this case as it may confuse users?

Reply:
Makes sense. Will amend.